plugin/src/main/java/io/jenkins/plugins/remotingkafka/KubernetesUtils.java

```diff
@@ -0,0 +1,14 @@
+package io.jenkins.plugins.remotingkafka;
+
+import io.fabric8.kubernetes.api.model.ServicePort;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+import java.util.List;
+
+public class KubernetesUtils {
```

Comment:
It is best not to name things "Utils"...maybe call this `KubernetesQuery`?

Reply:
@afalko Why should we not name things "Utils"?

Reply:
"Utils", "Helper" aren't good at describing what the class is doing and its purpose. This was a tough habit for me to kick as well.

Here's a better elucidated argument: https://gsferreira.com/archive/2017/09/please-don-t-name-it-utils/

Reply:
@afalko is this a preference or a general programming practice? TBH I am still seeing in a lot of places that people use helpers, utils, ....

Reply:
It is a general programming practice. Using "helpers", "utils" is considered lazy - at least according to a lot of the people I've worked with.